src/components/Form/FormUtils.js

```diff
@@ -75,7 +75,7 @@ export const createDiscountPayload = (form) => {
         case "Payment Method Discount": {
             return {
                 ...discountPayload,
-                "payment_method": form["Discount Rules"]["Payment Method"],
+                "payment_method": form["Discount Rules"]["Payment Method"].toLowerCase(),
```

Comment:
Is `International Credit Card` option handled properly?

Reply:
i think so... but for some reason none of the payment method discounts are being triggered